XenModel/Messages.resx

```diff
@@ -8850,6 +8892,9 @@ It is strongly recommended that you Cancel and apply the latest version of the p
   <data name="NOT_AGILE_VM_HAS_VGPU" xml:space="preserve">
     <value>The VM has one or more virtual GPUs. Restart cannot be guaranteed.</value>
   </data>
+  <data name="NOT_AGILE_VM_HAS_VUSB" xml:space="preserve">
+    <value>The VM has one or more virtual USBs. Restart cannot be guaranteed.</value>

```

Comment:
_The VM has one or more USB devices attached. Restart cannot be guaranteed._

Reply:
Thanks.